hazelcast/src/main/resources/hazelcast-config-3.7.xsd

```diff
@@ -2749,6 +2749,16 @@
                     </xs:documentation>
                 </xs:annotation>
             </xs:element>
+            <xs:element name="store-count" type="xs:unsignedInt" minOccurs="0" maxOccurs="1">
+                <xs:annotation>
+                    <xs:documentation>
+                        Number of hot-restart stores per Hazelcast instance. Each partition thread is assigned
```

Comment:
I would not remove what's there right now. quite opposite actually. I'd add something like: "It's a trade-off between write speed and start time ....The more stores you have.....". To give some idea what impact it might have.